libs/langchain-community/src/vectorstores/pgvector.ts

```diff
@@ -506,22 +521,23 @@ export class PGVectorStore extends VectorStore {
    */
   async ensureCollectionTableInDatabase(): Promise<void> {
     try {
-      await this.pool.query(`
+      const queryString = `
         CREATE TABLE IF NOT EXISTS ${this.collectionTableName} (
           uuid uuid NOT NULL DEFAULT uuid_generate_v4() PRIMARY KEY,
           name character varying,
           cmetadata jsonb
         );
 
-        ALTER TABLE ${this.tableName}
+        ALTER TABLE ${this.computedTableName}
           ADD COLUMN collection_id uuid;
 
-        ALTER TABLE ${this.tableName}
-          ADD CONSTRAINT ${this.tableName}_collection_id_fkey
+        ALTER TABLE ${this.computedTableName}
+          ADD CONSTRAINT ${this.computedTableName}_collection_id_fkey
```

Comment:
Shoud be this.tableName in the constraint identifier. Think this would throw cause of quotes?

Reply:
Yes that would throw due to the quotes. Just pushed the change using this.tableName instead